### PR TITLE
enable functionalTest in nightly build

### PR DESCRIPTION
**Pull request by klawani**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

enable backend functionalTest for Nightly build

**Does this PR introduce a breaking change?** (check one with "x")

- [ ] Yes
- [ x] No

### CODE REVIEW

**Comment by pragnesh-patel** [comment]
Its imperative these don't get disabled in future

File: Jenkinsfile_nightly [hunk 1]
```diff
@@ -24,4 +24,5 @@ withNightlyPipeline(type, product, component) {
     env.Rules = params.SecurityRules
     enableSlackNotifications(channel)
     enableSecurityScan()
```
Comment (pragnesh-patel) [inline Jenkinsfile_nightly:26]: Swapping temporarily the order of functional tests and security scan will enable us to run the functional tests first

Comment (klawani) [reply Jenkinsfile_nightly:26]: does the order matter ?

Comment (pragnesh-patel) [reply Jenkinsfile_nightly:26]: We could run the functional tests first if security is failing

**Pull request merged**